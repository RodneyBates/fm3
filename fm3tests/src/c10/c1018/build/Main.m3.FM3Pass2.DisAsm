 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L1(16_1),[1,0])
 ItkBlockDeclListLt(L1(16_1),[5,2])
 ItkConstDeclLt([5,8])
 ItkDeclNo(D1,[5,8])
 ItkConstDeclColon([5,14],FALSE)
 ItkDeclTypeAbsent()
 ItkConstDeclEquals([5,14])
 ItkInvalidRef([5,16])
 ItkConstDeclRt([5,8])
 ItkBlockDeclListRt(L1(16_1),[5,2])
 ItkModuleBegin(L1(16_1),[7,2])
 ItkStmtListLt(L0(16_0),[8,2])
 ItkStmtListRt(L0(16_0),[8,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
