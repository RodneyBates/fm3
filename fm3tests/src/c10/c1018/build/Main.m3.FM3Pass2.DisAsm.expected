   245 16_00000000000000d4                   212 ItkEOF()
   243 16_00000000000000d5                   213 ItkLeftEnd()
   241 16_00000000000000e1                   225 ItkModuleLt(L1(16_1),[1,0])
   236 16_00000000000000f0                   240 ItkDeclScopeLt(L1(16_1))
   233 16_00000000000000f2                   242 ItkOpenScopeLt(L1(16_1))
   230 16_0000000000000254                   596 ItkOpenDeclListLt(L6(16_6),[9,2])
   225 16_0000000000000143                   323 ItkConstDeclLt([9,8])
   221 16_00000000000000f7                   247 ItkDeclNo(D6,[9,8])
   216 16_0000000000000147                   327 ItkConstDeclType([9,15])
   212 16_00000000000001e4                   484 ItkDeclTypeAbsent([9,15])
   208 16_000000000000014a                   330 ItkConstDeclValue([9,15])
   204 16_0000000000000146                   326 ItkConstDeclRt([9,8])
   200 16_0000000000000258                   600 ItkOpenDeclListSep(L1(16_1),[9,2])
   195 16_0000000000000143                   323 ItkConstDeclLt([10,8])
   191 16_00000000000000f7                   247 ItkDeclNo(D7,[10,8])
   186 16_0000000000000147                   327 ItkConstDeclType([10,15])
   182 16_00000000000001e4                   484 ItkDeclTypeAbsent([10,15])
   178 16_000000000000014a                   330 ItkConstDeclValue([10,15])
   174 16_0000000000000146                   326 ItkConstDeclRt([10,8])
   170 16_0000000000000258                   600 ItkOpenDeclListSep(L2(16_2),[9,2])
   165 16_0000000000000143                   323 ItkConstDeclLt([11,8])
   161 16_00000000000000f7                   247 ItkDeclNo(D8,[11,8])
   156 16_0000000000000147                   327 ItkConstDeclType([11,15])
   152 16_00000000000001e4                   484 ItkDeclTypeAbsent([11,15])
   148 16_000000000000014a                   330 ItkConstDeclValue([11,15])
   144 16_0000000000000146                   326 ItkConstDeclRt([11,8])
   140 16_0000000000000258                   600 ItkOpenDeclListSep(L3(16_3),[9,2])
   135 16_0000000000000143                   323 ItkConstDeclLt([13,8])
   131 16_00000000000000f7                   247 ItkDeclNo(D9,[13,8])
   126 16_0000000000000147                   327 ItkConstDeclType([13,15])
   122 16_00000000000001e4                   484 ItkDeclTypeAbsent([13,15])
   118 16_000000000000014a                   330 ItkConstDeclValue([13,15])
   114 16_00000000000000f8                   248 ItkDefTopExprNo(L3(16_3),[13,17])
   109 16_0000000000000146                   326 ItkConstDeclRt([13,8])
   105 16_0000000000000258                   600 ItkOpenDeclListSep(L4(16_4),[9,2])
   100 16_0000000000000143                   323 ItkConstDeclLt([14,8])
    96 16_00000000000000f7                   247 ItkDeclNo(D10,[14,8])
    91 16_0000000000000147                   327 ItkConstDeclType([14,15])
    87 16_00000000000001e4                   484 ItkDeclTypeAbsent([14,15])
    83 16_000000000000014a                   330 ItkConstDeclValue([14,15])
    79 16_00000000000000f8                   248 ItkDefTopExprNo(L2(16_2),[14,17])
    74 16_0000000000000146                   326 ItkConstDeclRt([14,8])
    70 16_0000000000000258                   600 ItkOpenDeclListSep(L5(16_5),[9,2])
    65 16_0000000000000143                   323 ItkConstDeclLt([15,8])
    61 16_00000000000000f7                   247 ItkDeclNo(D11,[15,8])
    56 16_0000000000000147                   327 ItkConstDeclType([15,15])
    52 16_00000000000001e4                   484 ItkDeclTypeAbsent([15,15])
    48 16_000000000000014a                   330 ItkConstDeclValue([15,15])
    44 16_00000000000000f8                   248 ItkDefTopExprNo(L1(16_1),[15,17])
    39 16_0000000000000146                   326 ItkConstDeclRt([15,8])
    35 16_0000000000000257                   599 ItkOpenDeclListRt(L6(16_6),[9,2])
    30 16_00000000000000e5                   229 ItkModuleBegin(L1(16_1),[17,2])
    25 16_000000000000025c                   604 ItkStmtListLt(L0(16_0),[18,2])
    20 16_000000000000025f                   607 ItkStmtListRt(L0(16_0),[18,2])
    15 16_00000000000000f3                   243 ItkOpenScopeRt(L1(16_1))
    12 16_00000000000000f1                   241 ItkDeclScopeRt(L1(16_1))
     9 16_00000000000000e4                   228 ItkModuleRt(L1(16_1),[1,0])
     4 16_00000000000000d6                   214 ItkRightEnd()
     2 16_00000000000000d3                   211 ItkBOF()
     0
